### PR TITLE
Add files with completed tasks

### PR DESCRIPTION
**Pull request by AndyRem1x**

### CODE REVIEW

File: funcobjtask/fntask1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def func(arg1, arg2, arg3, *args, arg4=5, **argsdict):
+    test_var_1 = 100
+    test_var_2 = "Test variable"
+    test_var_3 = 500
+    test_var_4 = "Some text"
+    test_var_5 = 1
+    return len(locals()) - len([var for var in locals() if var.startswith("arg")])  # excluding the arguments of function
```
Comment (nakutnyi) [inline funcobjtask/fntask1.py:7]: Why do we exclude arguments? They are also local variables (they become local variables when you call a function)

File: funcobjtask/fntask3.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+def choose_func(nums: list, func1, func2):
+    for num in nums:
+        if num < 0:
+            return func2(nums)
```
Comment (nakutnyi) [inline funcobjtask/fntask3.py:4]: This solution is even simpler than mine. Good work

File: funcobjtask/fntask1.py (hunk 1 above)
Comment (AndyRem1x) [reply funcobjtask/fntask1.py:7]: It was mentioned in task that you need to to detect the number of local variables which are declared in a function.

Comment (AndyRem1x) [reply funcobjtask/fntask1.py:7]: So I thought only about variables that are declared in function's body

**Pull request merged**